Unity-Technologies/com.unity.multiplayer.samples.coop
# fix: join code not displayed for clients using Unity Relay

### Description (*)

This PR makes the Unity Relay join code accessible to clients after joining.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-2009](https://jira.unity3d.com/browse/MTT-2009)
### Manual testing scenarios

1. Start a game as a host using Unity Relay
2. Join the game as a client
3. See that now the join code is properly displayed on both client and host
### Questions or comments

### Contribution checklist
